plugins/python-build/scripts/add_miniforge.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+'Adds the latest miniforge and mambaforge releases. Assumes Python 3.10.'
+from pathlib import Path
+import logging
+import os
+import string
+
+import requests
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=os.environ.get('LOGLEVEL', 'INFO'))
+
+MINIFORGE_REPO = 'conda-forge/miniforge'
+PYTHON_VERSION = '310'
+
+install_script_fmt = """
+case "$(anaconda_architecture 2>/dev/null || true)" in
+{install_lines}
+* )
+  {{ echo
+    colorize 1 "ERROR"
+    echo ": The binary distribution of {flavor} is not available for $(anaconda_architecture 2>/dev/null || true)."
+    echo
+  }} >&2
+  exit 1
+  ;;
+esac
+""".lstrip()
+
+install_line_fmt = """
+"{os}-{arch}" )
+  install_script "{filename}" "{url}#{sha}" "miniconda" verify_py{py_version}
+  ;;
+""".strip()
+
+here = Path(__file__).resolve()
+out_dir: Path = here.parent.parent / "share" / "python-build"
+
+latest = requests.get(f'https://api.github.com/repos/{MINIFORGE_REPO}/releases').json()[0]
```

Comment:
So, it only adds the _latest_ release at the moment? How do we know when to run it then?

Maybe it's better to add all releases that are not in Pyenv yet?

Reply:
> How do we know when to run it then?

When someone wants a new version 😃 

There are actually fewer missing versions than I initially thought, so I'll change this to backfill.